external-import/ransomwarelive/src/lib/ransomConn.py

```diff
@@ -605,21 +621,22 @@ def stix_object_generator(self, item, group_data):
             and len(item.get("country", "Four")) < 4
         ):
 
-            country_stix_id = self.opencti_location_check(item.get("country"))
+            country_name = item.get("country")
+            country_stix_id = self.opencti_location_check(country_name)
```

Comment:
Suggestion to reduce duplication and remove unused argument

```suggestion
            country_stix_id = self.opencti_location_check(country_name) or pycti.Location.generate_id(country_name, "Country")
            location3 = Location(
                    id=country_stix_id,
                    name=country_name,
                    country=item.get("country"),
                    created_by_ref=self.author.get("id"),
                    object_marking_refs=[self.marking.get("id")],
                )
            bundle.append(location3)    
```